### PR TITLE
Data overwrite and README

### PR DESCRIPTION
**Pull request by chris-kim-agent1**
This PR includes the following updates:
1. README - Added an overview paragraph and description for available dataset(s).
2. Removed initial `ukraine/ukraine_russia_conflict_iocs.csv` file. Created 2 new files that aligns with MISP and internally agreed format. `ukraine_russia_legitimate_iocs.csv` and `ukraine_russia_malicous_suspicious_iocs.csv`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+that can cause harm to businesses and innocent users. TIG recommends blocking traffic from network indicators described 
+in this file.
+
+###### Schema Table
+| Field       | Description |
+| ----------- | ----------- |
+| type        | The data type of the IOC. Possible options: domain, ip, hostname, host, url, sha256, email       |
+| indicator   | Also known as an IOC, this analysis artifact is a piece of forensic data related to online activities regarding the Russia-Ukraine conflict.|
+| context     | Descriptive labels that explain the nature of the IOC |
+| source      | The lead we used to find the IOC. If the IOC is derived from a public source, this field will contain a reference link. |
```
Comment (reneeburton) [inline README.md:26]: i think this source is generally not the lead; but a reference we used in checking the lead. i'd make this references. and then include notes.  for context we'll have classification.

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+is disclosed [privately](https://www.infoblox.com/products/threat-intelligence/) to paying customers, TIG shares 
+indicators of compromise (IOCs) related to threats that are of high interest to the cyber security community through
+this public repository. The following contains descriptions about the contents of each dataset (i.e data folder). All files
+are csv formatted and [MISP](https://www.misp-project.org/) compatible.
+
+
+### ukraine
+This folder contains IOCs related to the Russia-Ukraine conflict. Majority of the content has been organically generated
+using both internal and private data. The `ukraine_russia_legitimate_iocs.csv` file contains confirmed IOCs that are
+managed by legitimate organizations. This includes domains belonging to well-known donation sites providing support to
```
Comment (reneeburton) [inline README.md:14]: I would say legitimate file "contains confirmed indicators that, at the time of review, were not evidently associated with malicious activity. The related websites did not show indications of hosted malware or fraudulent behavior, but may still host content that is not wanted by some users."   (we have dating sites, e.g., in there)

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+Ukrainian civilians, or even newly created support programs operated by entities positively acknowledged by the 
+online community. Oppositely, `ukraine_russia_malicous_suspicious_iocs.csv` contains malicious and suspicious IOCs
+that can cause harm to businesses and innocent users. TIG recommends blocking traffic from network indicators described
```
Comment (reneeburton) [inline README.md:17]: Here i'd say `Infoblox recommends..`

**Comment by chris-kim-agent1** [comment]
@reneeburton I have edited the texts based on your suggestions. I will update the csv files once everything has been finalized internally.

**Review by reneeburton — APPROVED** [review]

**Pull request merged**